ui/apps/platform/src/services/DeploymentsService.ts

```diff
@@ -10,15 +9,14 @@ import {
     orchestratorComponentOption,
 } from 'Containers/Navigation/OrchestratorComponentsToggle';
 import axios from './instance';
-import { deployment as deploymentSchema } from './schemas';
```

Comment:
Can you double check whether the following exports from schemas.js now lack any references:
* deployment
* deploymentDetail
* alert
* alerts